architecture/core_concepts/routes.adoc

```diff
@@ -314,6 +320,47 @@ they are unique on the machine. These ports will not be exposed externally.
 
 The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
 
+[[load-balancing]]
+== Load-balancing Strategy
+
+When a route has multiple endpoints, HAProxy distributes requests to the route
+among the endpoints based on the selected load-balancing strategy. This only
+applies when no persistence information is available, or when a connection is
+redispatched to another server. See the discussion on
```

Comment:
"what does re-dispatched to another server" mean? Do you mean when a back-end goes away and it needs to pick a new one?